src/Camera.cpp

```
@@ -0,0 +1,66 @@
+#include "../include/Camera.h"
+
+Camera::Camera() : Device() {
+	resolution = {1920, 1080};
```

Cppcheck (2.10-2)

[performance] Variable 'resolution' is assigned in constructor body. Consider performing initialization in initialization list.

(useInitializationList)